### PR TITLE
css fixed for All posts

### LINKED ISSUE #81: [BUG] Issue with the title alignment
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

The "All Posts" title is not aligned with the image.

### Expected Behavior

All Posts should be aligned with the image box below.

### Reproduction URL

https://wanderlust-beta.vercel.app

### Reproduction steps

```bash
1. Go to 'https://wanderlust-beta.vercel.app'
2. Scroll down to 'All Posts'
3. See error
```

### Screenshots

```bash
![DESCRIPTION](LINK.png)
https://cdn.discordapp.com/attachments/1184178942459007136/1184902389359181904/image.png?ex=658da958&is=657b3458&hm=7110333f0ac297f5689172ad502f5bf267f36af41aeea68be52c63f1dff2376f&
```

### Logs

_No response_

### Browsers

_No response_

### OS

_No response_

### PR DESCRIPTION
**Pull request by Deblin798**
## Summary

All posts alignment fixed

## Description

'All posts' title and the posts were not aligned properly, added CSS to fix the alignment according to figma design

## Images

image

## Issue(s) Addressed

_Enter the issue number of the bug(s) that this PR fixes_

Closes #81 

## Prerequisites

- [ ] Have you followed all the [CONTRIBUTING GUIDELINES](https://github.com/krishnaacharyaa/wanderlust/blob/main/.github/CONTRIBUTING.md#guidelines-for-contributions)? Yes

### CODE REVIEW

File: frontend/src/pages/home-page.tsx [hunk 1]
```diff
@@ -53,7 +53,7 @@ function HomePage() {
       <div className="mx-4 md:mx-8 lg:mx-16">
         <BlogFeed />
         <h1 className="text-2xl font-semibold dark:text-dark-primary">All Posts</h1>
-        <div className="flex flex-wrap">
+        <div className="flex flex-wrap" style={{ marginLeft: '-15px' }}>
```
Comment (shmbajaj) [inline frontend/src/pages/home-page.tsx:56]: Please use tailwind-css.

**Review by shmbajaj — CHANGES_REQUESTED** [review]
Also share before and after screenshots to understand changes/difference.

**Comment by krishnaacharyaa** [comment]
@Deblin798 this isn't the right way of fixing, you need to figure out why that is happening and you need to remove that in the blogs post and also skeleton, it is not hardcoded fix that we are expecting :)
You can refer the previous PR and refer it and solve the comments, that will better place to start

**Comment by Deblin798** [comment]
@krishnaacharyaa @shmbajaj updated my PR

checkpoints - 
1. Used Tailwind CSS for the inline changes
2. change made - instead of applying margin-4 to all the sides of the post card, used specific sides where margin is supposed to be added
3. attaching before and after -
Before - 
image
After - 
image

**Comment by krishnaacharyaa** [comment]
Thank you for the quick update, and taking time in understanding the requirement and acting on it :)

p.s: Kindly follow the commit guidelines from the next time, we expect usage of prefixes which roughly tells us what the commit is about, you can refer community guidelines or any other merged PR for the same

**Review by krishnaacharyaa — APPROVED** [review]
LGTM

**Pull request merged**